specification/gp-connect-prescriptions-management-fhir.yaml

```diff
@@ -125,15 +125,43 @@ info:
 
     * authenticated with NHS login to P9 identity verification level
 
-    The API uses Open ID Connect to authenticate the end user and OAuth 2.0 to authorise the calling system. It supports the following security pattern using NHS login: XXXXX
+    The API uses Open ID Connect to authenticate the end user and OAuth 2.0 to authorise the calling system. It supports the following security pattern using NHS login: 
+    
+    * [User-restricted RESTful APIs - NHS login separate authentication and authorisation](https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation/user-restricted-restful-apis-nhs-login-separate-authentication-and-authorisation)
 
     ## Environments and testing
     
     |Environment|Base URL|
     |-----------|--------|
     |Sandbox|`https://sandbox.api.service.nhs.uk/gp-connect-prescriptions-management-fhir`|
```

Comment:
I'll need to be check this, will probably bring it up in the design review. When I look at EPS they don't have FHIR/R4 in the URL but have it for each endpoint (it might be an APIM guideline).